environment/swarmball_env.py

```diff
@@ -4,17 +4,24 @@
 from .simulation.simulation import SwarmBallSimulation
 
 class SwarmBall(gym.Env):
-    def __init__(self):
-        self.sim = SwarmBallSimulation()
+    def __init__(self, acc_factor=0.25, thresh_amount=3, **kwargs):
+        self.sim = SwarmBallSimulation(**kwargs)
+        self.thresholds = [[] for _ in range(thresh_amount)]
+        self.thresh_vel = [0,0,0]
```

Comment:
Spacje po przecinkach. You've just been PEP8'ed!

Reply:
I czy jeśli będziemy mieli więcej niż 3 klastry to czy ta lista nie powinna być odpowienio większa? Sugeruję uzależnić to od thresh_amount

Reply:
Fair point. To jedna z tych sytuacji "Zmienie to na ładnie tuż przed PR" albo najwyraźniej jak się jest łosiem to nawet w trakcie PR.

Reply:
Fair point. To jedna z tych sytuacji "Ogarnę na ładnie przed PR" albo najwyraźniej w trakcie PR jak się jest łosiem.

Reply:
Zdarza się :D Częściej z resztą niż ktokolwiek by chciał. Od tego jest review